pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchurl, kubernetes }:
+let
+  arch = if stdenv.isLinux
```

Comment:
To be honest it seems a bit of a pain (and I need to figure out what's going on a little better). I modeled this on `minikube` which seemed similar, so hopefully it's ok for now "as is".

I would be keen to make the package available as it is, but will happily work on improving it to build from source instead.